.changeset/fruity-laws-rule.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": tree
+---
+Defaulted identifier fields on unhydrated nodes are now enumerable
```

Comment:
Should we add "unhydrated" to the vale dictionary?

Reply:
I'm not sure. I don't think unhydrated is a good term, and we should probably move to something else to better along with the rest of fluid. I think deprecating the term unhydrated and moving to use [Detached](https://fluidframework.com/docs/api/container-definitions/attachstate-enum#detached-enummember) instead would be better.